### DISCUSSION TITLE
Unable to run the file on Debian 10

### DISCUSSION THREAD

**Discussion by reyensalada** [kaiserdj/Darkorbit-client#34] [Q&A] [answered]
Hey, I want to use the.appimage file on my Debian 10 laptop and I can't access it, at first if I could run the file but after deleting it by mistake, please if someone fixes this, I would appreciate them too much:( 

PS: Sorry if I don't understand correctly, I'm using google translate

**Comment by kaiserdj** [comment]
try this: 

https://user-images.githubusercontent.com/5487950/126153333-8b7ab2ce-93c6-4df5-8e24-f07cf66439dd.mp4

**Reply by reyensalada** [reply]
This happens to me :( 

https://drive.google.com/file/d/1-tJmVCz1XG7ZzH5uj1HuMaxQPtFlV8ME/view?usp=sharing

**Reply by kaiserdj** [reply]
try with this:
Open a terminal
Change to the directory containing the AppImage, e.g., using cd <my directory>
Make the AppImage executable: chmod +x my.AppImage
Run the AppImage: ./my.AppImage

**Reply by reyensalada** [reply]
Sorry for the delay, so far I have had time to test it. 
It turns out that I keep getting errors, I have investigated about it and it seems that it is an error that occurs in chrome.

here is the video of the error:

https://drive.google.com/file/d/1tb3ePVwIOq5x5pzDFghXILyZObfNZznf/view?usp=sharing

**Reply by reyensalada** [reply]
LOOOOOOLLLL BRO, I was able to solve the problem, I just had to update to Debian 11, also thanks for the help you have given me, I plan to upload a tutorial to youtube about your file so that others in my situation can play it wonderful game.

**Comment by reyensalada** [comment, marked as answer]
The answer to remove this error is to upgrade to Debian 11, done, thread solved
